pkgs/development/python-modules/invoicegenerator/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, babel
+, buildPythonPackage
+, fetchFromGitHub
+, pillow
+, pythonOlder
+, reportlab
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "invoicegenerator";
+  version = "1.1.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "by-cx";
+    repo = "invoicegenerator";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-NrsDeABxaKaEF7RdPH3Kj0eyzgm087qgv0otiyGkYgQ=";
+  };
+
+  # Remove dependency: qrplatba
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "\"qrplatba>=0.3.3\"," ""
+  '';
```

Comment:
The feature is not nessesary fro the package, it is also missing in the current master branch. Same with (xml unittest)

Reply:
It seems to me that this package is called when `pdf.gen("invoice.pdf", generate_qr_code=True)`. (https://github.com/by-cx/InvoiceGenerator/blob/a0a33798ddcc84a69282c4c5d7429b880d7b53e2/InvoiceGenerator/pdf.py#L134-L135)
However, I am not sure since I have not used this package.

Reply:
Yes thats true, but implement another package would be too much work for me, fell free to import another package. As long as "generate_qr_code=False" this pkg fully works.

Reply:
This is not part of extras in setup.py or has a real feature detection if the package exsists or not.